sort.rb

```diff
@@ -0,0 +1,25 @@
+
+def sort(array) 
+  if array == nil || array.length <= 1
+    return array # nothing to sort
+  end
+
+  index = 0
+  swapped = true
+  while index < array.length && swapped
+    swapped = false
+    other_index = index + 1
+    while other_index < array.length
+      if array[index] > array[other_index]
+        temp = array[index]
+        array[index] = array[other_index]
+        array[other_index] = temp
```

Comment:
The following might help with the readability of this block.

```suggestion
       [array[index], array[other_index]]= [array[other_index], array[index]]
```